libs/ui/elements/src/lib/thumbnail/thumbnail.component.html

```diff
@@ -1,7 +1,9 @@
 <div class="h-full w-full relative flex-shrink-0 overflow-hidden">
   <img
+    #imageElement
     *ngIf="imgUrl"
-    class="relative h-full w-full object-cover object-center"
+    class="relative h-full w-full object-center"
+    [style.object-fit]="isPlaceholder ? 'scale-down' : 'cover'"
```

Comment:
I guess you will still handle the `scale-down` for the `fallbackUrl` as well, so that the placeholder in the fallback case appears the same?

Reply:
yes, that's what I was struggling with; eventually I just removed the fallback directive and did everything in the thumbnail component which made things simpler (see latest commits)